recipe/meta.yaml

```diff
@@ -33,13 +35,12 @@ test:
     - pip check
 
 about:
-  home: https://gitlab.com/documatt/sphinx-reredirects
-  license: BSD-3-Clause
-  license_family: BSD
+  home: https://documatt.com/sphinx-reredirects
+  license: MIT
```

Comment:
It is worth noting that the author seems to have changed the license without consent of the contributors and the BSD license has been removed. I think that technically the BSD license must remain, as all prior code contributed is licensed under the BSD3.